collection-spec/collection-spec.md

```diff
@@ -24,7 +24,7 @@ Implementations are encouraged, however, as good effort will be made to not chan
 | Element      | Type              | Description                                                  |
 | ------------ | ----------------- | ------------------------------------------------------------ |
 | stac_version | string            | **REQUIRED.** The STAC version the collection implements.    |
-| id           | string            | **REQUIRED.** Identifier for the collection that is unique across the provider. |
+| id           | string            | **REQUIRED.** Identifier for the collection that is unique across the provider. The use of URI reserved characters such as `:` or `/` is discouraged since this will result in [percented encoded](https://tools.ietf.org/html/rfc3986#section-2) STAC endpoints. |
```

Comment:
This should also be added to the catalog specification.

Additionally, in all three specs I'd write `STAC API` or `STAC API endpoints` instead of just `STAC endpoints`.

Reply:
Done.